src/kv/serialise_entry_blit.h

```diff
@@ -20,6 +20,19 @@ namespace kv::serialisers
       {
         return SerialisedEntry(t.begin(), t.end());
       }
+      else if constexpr (std::is_same_v<
+                           T,
+                           std::pair<std::string, std::vector<uint8_t>>>)
+      {
+        const auto& [str, vec] = t;
+        auto size = sizeof(size_t) + str.size() + sizeof(size_t) + vec.size();
+        SerialisedEntry s(size);
+        auto data = s.data();
+        serialized::write(data, size, str);
+        serialized::write(data, size, vec.size());
+        serialized::write(data, size, vec.data(), vec.size());
+        return s;
+      }
```

Comment:
Non-critical, but it should be possible to do this generically. Something like:

```
if constexpr (nonstd::is_specialization<T, std::pair>::value)
{
  auto size = serialized::size(t.first) + serialized::size(t.second);
  SerialisedEntry s(size);
  auto data = s.data();
  serialized::write(data, size, t.first);
  serialized::write(data, size, t.second);
  return s;
}
```

This doesn't quite work because `serialized::size` doesn't exist (we manually calculate the size before calling `write` everywhere), and `serialized::write(..., ..., const std::vector<uint8_t>& v)` doesn't do size-prefixing, so we do it externally, but if we could fix those this would be more general.